src/pymor/algorithms/rules.py

```diff
@@ -382,7 +391,9 @@ def get_children(cls, obj):
                 v = getattr(obj, k)
                 if (isinstance(v, Operator)
                         or isinstance(v, dict) and all(isinstance(vv, Operator) or vv is None for vv in v.values())
-                        or isinstance(v, (list, tuple)) and all(isinstance(vv, Operator) or vv is None for vv in v)):
+                        or isinstance(v, (list, tuple)) and all(isinstance(vv, Operator) or vv is None for vv in v)
+                        or isinstance(v, np.ndarray) and v.dtype == object and v.ndim >= 1
```

Comment:
Why is `v.ndim >= 1` needed? Scalar arrays should work as well?

Reply:
Agree